src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -11,6 +11,7 @@
 static constexpr size_t simd = 16;
 static constexpr size_t min_quantize_grp_size = 32;
 static constexpr size_t min_slm_size = 256;
+static constexpr size_t min_opt_num_threads = 3072;
```

Comment:
This is machine specific.. we'll need to use some more insight from the machine spec. E.g, did you check on ADL?

Reply:
updated as discussion on offline